spec/mem/interface.md

```diff
@@ -59,6 +59,9 @@ pub enum AllocationKind {
     Function,
 }
 
+/// The ID of each function call (i.e stack frame)
+type CallId = Int;
```

Comment:
Let's not have two types, `CallId` and `FrameId`. Instead, maybe add a `type FrameId;` to the `trait Memory`?